tests/unit/CMakeLists.txt

```diff
@@ -17,6 +21,11 @@ if(BUILD_TESTING) #set by Ctest. Also set in the integration build environment.
         endif()
     endif()
 
+    add_compile_options("-fpermissive")
```

Comment:
Do we really want to enable `-fpermissive`? I'd prefer to fix the warnings instead.

Reply:
no, not needed. Some leftovers from testing.